sw/device/lib/flash_ctrl.c

```diff
@@ -113,10 +115,41 @@ static int flash_write_internal(uint32_t addr, part_type_t part,
   return get_clr_err();
 }
 
+// address is assumed to be aligned to uint32_t
```

Comment:
Nit: can you make sure to capitalize comments and end them with punctuation?

Reply:
yes will do.